todo/tests/fixtures/label.py

```
@@ -0,0 +1,20 @@
+from bson import ObjectId
+from todo.models.label import LabelModel
```

Ruff (0.8.2)

`todo.models.label.LabelModel` imported but unused

Remove unused import: `todo.models.label.LabelModel`

(F401)